package.json

```diff
@@ -46,6 +46,9 @@
     "test": "vitest run --coverage",
     "test:watch": "vitest --coverage"
   },
+  "dependencies": {
+    "is-inside-container": "^1.0.0"
+  },
```

Comment:
As the only dependency, this seems wrong. I would look into how dependencies are bundled so when this package is installed, it doesn't cause other packages to also be installed.